entrypoints/content/pages/feed.tsx

```diff
@@ -1,10 +1,48 @@
-// Packages:
-import React from 'react'
+// Components:
+import { TOPICS } from 'constants/database/comments-and-replies'
+import Comment from '../components/secondary/comment/Comment'
+import { FieldValue } from 'firebase-admin/firestore'
+
+const SAMPLE_USER_INFO = {
+  name: 'Ben Holmes',
+  username: '@BenHolmesDev',
+  timePosted: '17h',
+  profileImgUrl: 'https://github.com/shadcn.png'
+}
+
+const SAMPLE_CONTENT = `Right when I heard Stephen first talk I had a gut feeling that it was him. 
```

Comment:
oh btw, 2 things for comment body
- we need to preserve new lines
- we need to do that without dangerouslySetInnerHTML (https://cheatsheetseries.owasp.org/cheatsheets/Cross_Site_Scripting_Prevention_Cheat_Sheet.html#framework-security)